Automattic/wp-calypso
# Add/resend email button

#### Proposed Changes

There should be an option to resend the verification email for unverified accounts. 

#### Testing Instructions

Note: this new notice is rendered on routes `/me/account` and `me/security/account-email` and can be tested at both locations. It's the same code running on each page. 

1. Create a new account and don't action the verification email. 
2. Navigate to `/me/account` and you should see a notice under "Email address" as shown:
![image](https://user-images.githubusercontent.com/6851384/175278696-e843b139-adcf-4005-bcdb-d83c9ee0d76c.png)
3. Click on "Resend email" and an email should be sent out. Don't verify though. Confirm positive feedback is shown:
![image](https://user-images.githubusercontent.com/6851384/175283314-9e5474e2-f72b-4ef1-8f1d-a12e21bdd57c.png)
4. Disable your network connection and then hit "Resend email" again"
5. Confirm negative feedback is shown:
![image](https://user-images.githubusercontent.com/6851384/175283349-0eb95c27-feb8-4dc0-818c-868300e63a9c.png)
6. Enable network and refresh the page.
7. Go to your Inbox and verify the email. 
8. Refresh the page and the notice should have disappeared. 


Related https://github.com/Automattic/wp-calypso/issues/64345
